src/main/java/uk/gov/pay/webhooks/message/apirepresentation/PaymentApiRepresentation.java

```diff
@@ -0,0 +1,68 @@
+package uk.gov.pay.webhooks.message.apirepresentation;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import uk.gov.pay.webhooks.ledger.model.LedgerTransaction;
+import uk.gov.pay.webhooks.ledger.model.SettlementSummary;
+
+import java.util.Map;
+import java.util.Optional;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)
+public record PaymentApiRepresentation(
+        String paymentId,
+        Long amount,
+        PaymentState state,
+        String returnUrl,
+        String description,
+        String reference,
+        String email,
+        String paymentProvider,
+        String createdDate,
```

Comment:
Yes, I agree this is a clunky type, but this is how it comes from the LedgerTransaction model: https://github.com/alphagov/pay-webhooks/blob/e098b1e95b7f5df1d3abdeadc26255ae2457a61e/src/main/java/uk/gov/pay/webhooks/ledger/model/LedgerTransaction.java#L126-L128 so seems weird to transform it for no reason?

Reply:
I see your point: it comes in as a string, it goes out as a string. Part of me wishes it would be of an appropriate type when it’s actually inside the microservice, so it’s at least pure and correct during the times it doesn’t have to interface with the real world. So I won’t complain if you leave it as a string.